src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/jaxrs/JaxRsContext.java

```diff
@@ -87,7 +85,7 @@ public void setRootPath(String rootPath) {
 	public String getApplicationPath() {
 		if (applicationPath == null) {
 			PsiClass applicationPathType = PsiTypeUtils.findType(javaCodeLensContext.getJavaProject(),
-					JAVAX_WS_RS_APPLICATIONPATH_ANNOTATION);
+					JaxRsConstants.JAVAX_WS_RS_APPLICATIONPATH_ANNOTATION);
```

Comment:
I see in some cases you remove `import static` and this affects code in the body of the class. Could you restore those imports?

Reply:
@turkeylurkey - I have done the changes as suggested. Please review.